### PR TITLE
Update register look

### PR DESCRIPTION
**Pull request by leomendoza123**

### CODE REVIEW

**Comment by leomendoza123** [comment]
[Update visibility settings text on 'Register'](https://trello.com/c/rzmRNjRw/6815-update-visibility-settings-text-on-register)

File: src/app/register/components/form-visibility/form-visibility.component.html [hunk 1]
```diff
@@ -73,4 +79,14 @@ <h3 class="mat-subheading-2" i18n="@@register.visibility">
       >Please choose a default visibility setting.</mat-error
     >
   </p>
+  <p>
```
Comment (DanielPalafox) [inline src/app/register/components/form-visibility/form-visibility.component.html:82]: p tag it is not necessary here

File: src/locale/properties/register/register.xx.properties [hunk 2]
```diff
@@ -74,3 +74,6 @@ register.moreInfo=X
 register.securityAndNotifications=X
 register.visibilityAndTerm=X
 register.visibilityRequired=X
+register.VisibilityPrivate2=X
```
Comment (DanielPalafox) [inline src/locale/properties/register/register.xx.properties:77]: The properties `register.VisibilityPrivate1`, `register.VisibilityTrusted1` and `register.VisibilityEveryone1` are missing here and also in  the `rl` and `lr` properties files

File: src/app/register/components/form-visibility/form-visibility.component.html (hunk 1 above)
Comment (amontenegro) [reply src/app/register/components/form-visibility/form-visibility.component.html:82]: Nice catch!!!

Comment (leomendoza123) [reply src/app/register/components/form-visibility/form-visibility.component.html:82]: is require for the margins 
image

**Pull request merged**